src/layout/Navbar/Navbar.tsx

```diff
@@ -1,19 +1,23 @@
-import { useContext, useEffect, useState } from "react";
+import { useEffect, useState } from "react";
 import { NavLink } from "react-router-dom";
-import { fetchUserData } from "services/api";
-import { UserContext } from "context/UserProvider";
 import styles from "./navbar.module.css";
 import Profile from "./Profile/Profile";
-
+import { GetUserResponse } from "types/apis";
 import logoImage from "assets/images/logo.svg";
+import { fetchGetSampleUsers } from "utils/hooks/useGetSampleData";
 
 function Navbar() {
-  const { id } = useContext(UserContext);
-  const [user, setUser] = useState(null);
+  const [user, setUser] = useState<GetUserResponse>();
```

Comment:
여기도 이렇게 표현되면 좋을 것 같아요!
```suggestion
  const [user, setUser] = useState<User>();
```